source/administration/mmctl-cli-tool.rst

```diff
@@ -2352,6 +2355,47 @@ mmctl permissions add
    --local                       allows communicating with the server through a unix socket
    --strict                      will only run commands if the mmctl version matches the server one
 
+mmctl permissions role assign
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**Description**
+
+Assign users to a role by username (only available in Enterprise Edition E10 and E20).
+
+**Format**
+
+.. code-block:: sh
+
+  mmctl permissions role assign [role_name] [username...] [flags]
+
+**Examples**
+
+.. code-block:: sh
+
+    # Assign users with usernames 'john.doe' and 'jane.doe' to the role named 'system_admin'.
```

Comment:
I really like the addition of this context. One suggestion I have is to remove it from the code block and have it as normal text so it's more visually obvious. Thoughts?

Reply:
Content in code blocks are easy to copy and paste directly. My recommendation is to limit code blocks to only code that copies and pastes successfully for the user, and to split out great context like this as normal text.

Reply:
5/5 on limiting code blocks to that and splitting out the context and code. I do see the other instances of this and I think it's early enough to fix them and then going forward separate context and code?

Reply:
Based on above, let's leave this as-is being that it matches the current format and then look at changing it post-release in a new PR.

Reply:
Great! This update would be a great addition to the Style Guide. Once included, we can perform a sweep of the current docs to adjust and align to the Style Guide. Thoughts?

Reply:
I'm 5/5 on that! Will add it to our list. :)